docs/accordproject-concepts.md

```diff
@@ -5,31 +5,31 @@ title: Key Concepts
 
 ## What is an Accord Project Template?
 
-An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements: 
+An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements:
 
 - **The Template Text**: the natural language of the template
-- **The Template Model**: the data model that backs the template 
+- **The Template Model**: the data model that backs the template
 - **The Template Logic**: the executable business logic for the template
```

Comment:
These could be links to Cicero, Concerto, and Ergo?

Reply:
> These could be links to Cicero, Concerto, and Ergo?
> 
> Also, one of the `the`'s could be removed, preferably the first ones to make it
> 
> ```
> Template Text
> Template Model
> Template Logic
> ```

I think the first `The` could be removed.

The pointers to Cicero / Ergo / Concerto is given later in that same page, the idea here is to introduce the _concepts_ not the implementation of those concepts.